main.go

```diff
@@ -2,28 +2,51 @@ package main
 
 import (
 	"fmt"
+	"io/ioutil"
 	"os"
 	"strings"
 
+	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 	yaml "gopkg.in/yaml.v1"
 	"k8s.io/helm/pkg/helm"
 	"k8s.io/helm/pkg/strvals"
 )
 
+type cmdFlags struct {
+	cliValues       []string
+	resetValues     bool
+	templateNumbers []string
```

Comment:
Hi @edwardstudy 
`templateNumbers ` is no longer used , this can be removed. Thanks!